packages/fxa-payments-server/src/lib/test-utils.tsx

```diff
@@ -120,6 +120,14 @@ const MockStripeElement = ({ testid }: { testid: string }) =>
       );
     }
   };
+  // Ensure we have a test ID to identify which kind of Stripe
+  // element this is, later on. Using Object.defineProperty because
+  // a simple mockElement.testid = testid makes TypeScript angry
+  return Object.defineProperty(mockElement, 'testid', {
```

Comment:
This is kind of a weird hack, but it lets us verify that a `CardElement` was indeed passed through all the plumbing down to the `onSubmit` callback over in the payment update form.